simplerunner.lua

```diff
@@ -63,53 +109,276 @@ function runbench(name, count)
     if not run_iter then
         error("Missing benchmark function for '"..name.."'")
     end
+    
+    if loading_jitstats and jitstats.stats.starts > 0 then 
+        jitstats.print("Traces created while loading benchmark")
+        jitstats.reset()
+    end  
+end  
 
-    count = count or 1
+function runbench(name, count, scaling)
+    scaling = scaling or 1

```

Comment:
I'm not sure I see the point of this line.

Reply:
Its defaulting it to 1 or if the parameter is unset

Reply:
I can't see anywhere where the parameter is unset though?

Reply:
well the run_all_benchmarks call at the bottom doesn't pass it and i think passing a scaling value to run_all_benchmarks would be an override from a preconfigured scaling settings system we don't have

Reply:
I'm not sure I understand the sentence after "i think passing" -- can you rephrase?

Reply:
I think we should have our benchmark scaling values in a shared json file used by both our krun file and this

Reply:
I'm fine with that, at which point we wouldn't need this default?